data/NavItems.tsx

```diff
@@ -0,0 +1,10 @@
+export const NavItems = [
```

Comment:
rather than data rename the folder to constants, will give much more better idea

Reply:
@VineeTagarwaL-code , thanks for reviewing. Done with requested changes.